src/blocks/CaseStudyParallax/index.module.scss

```diff
@@ -0,0 +1,317 @@
+@use '@scss/common' as *;
+
+.cards {
+  position: relative;
+}
+
+.mainGutter {
+  position: relative;
+
+}
+
+.card {
+  position: relative;
+  padding-top:  calc(var(--base) * 4);
+
+  &.isFirst {
+    grid-column: 1/16;
+    grid-row: 1/1;
+  }
+
+  & .media {
+    margin-right: calc(var(--gutter-h) / -1);
+    overflow: hidden;
+
+    img {
+      right: -1px;
+
+      @include mid-break {
+        right: unset;
+      }
+    }
+
+    @include mid-break {
+      margin-right: 1px;
+      margin-left: 1px;
+    }
+  }
+
+  &:last-of-type {
+    padding-bottom: calc(var(--base) * 4);
+
+    @include mid-break {
+      padding-bottom:  0;
+    }
+  }
+}
+
+.stickyBlock {
+  position: sticky;
+  top: 18%;
+  z-index: 6;
+  grid-column: 1/16;
+  grid-row: 1/1;
+  pointer-events: none;
+  align-items: center;
+
+  @include large-break {
+    top: 23%;
+  }
+
+  @include mid-break {
+    display: none;
+  }
+}
+
+.stickyBlockItem {
+  grid-column: 1/9;
+  grid-row: 1/1;
+  opacity: 0;
+  transition: all 0.35s;
```

Comment:
Try doing something like this:

```
will-change: opacity, visibility;
transition: opacity 0.45s ease-in-out, visibility 0.45s ease-in-out;
```

instead of just `transition: all 0.35s`